lib/streamlit/connections/base_connection.py

```diff
@@ -164,7 +164,6 @@ def reset(self) -> None:
         >>> # Note: is_healthy() isn't a real method and is just shown for example here.
         >>> if not conn.is_healthy():
         ...     conn.reset()
-        ...
```

Comment:
Maybe replace with `>>>` to keep the line?

Reply:
if that works, maybe also do this for some of the other occurrences below?

Reply:
Yes, I still need to go in and correct the errors in linting. It's just that we had a release so I inquired if we needed to rerun the initial pass before I edited the PR. I can just start modifying this PR if we aren't going to rerun the initial script on the new docstrings that were introduced.

Reply:
Yeah, I think we can just reapply the formatting logic via:

```
ruff check --fix
ruff format
```